.npmignore

```diff
@@ -0,0 +1,3 @@
+.gitignore
```

Comment:
This is the culprit and on a first look I don't think it was intended to have this effect https://github.com/npm/fstream-npm/blob/master/fstream-npm.js#L331-L336